theano/gof/tests/test_vm.py

```diff
@@ -368,6 +370,55 @@ def check_storage(storage_map):
                             return [True, storage_map[o][0]]
             return [False, None]
 
-        assert check_storage(storage_map)[0]
-        assert len(set([id(v) for v in
-                        storage_map.values()])) < len(storage_map)
+        if config.memory_realloc:
```

Comment:
The test should enable it only for the test. Otherwise, it won't test it by default. If you rebase this PR, you can use a decorator like this:
```
@theano.configparser.change_flags(**{'vm.lazy': True})
```